src/test/java/gameframework/motion/blocking/MoveBlockerCheckerDefaultImplTest.java

```diff
@@ -79,6 +79,10 @@ public boolean isMovable() {
 			}
 		};
 	}
+	
+	private MoveBlockerMovable createMoveBlockerMovable() {
```

Comment:
please don't use `private` but `protected`. There is no reason to forbid your subclasses to adapt this method.